test/smoke/src/areas/positron/plots/plots.test.ts

```diff
@@ -15,8 +15,12 @@ test.use({
 	suiteId: __filename
 });
 
-// web bugs 4800 & 4804
-test.describe('Plots', () => {
+test.describe('Plots', {
+	annotation: [
+		{ type: 'issue', description: 'https://github.com/posit-dev/positron/issues/4800' },
+		{ type: 'issue', description: 'https://github.com/posit-dev/positron/issues/4804' }
+	]
+}, () => {
```

Comment:
Are these annotations correct?  I checked em out and they are both older closed issues,